SimPPS/PPSSimTrackProducer/python/SimTrackProducerForFullSim_cff.py

```diff
@@ -0,0 +1,34 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
@mundim , please confirm, that this variant of FullSim work only for PPS or you assume it can work for combination central CMS + PPS?

Reply:
It works for central CMS+PPS. There should be  a better way to do it though...